### PR TITLE
Feature/24423_jenkins_db_tests

### PR DESCRIPTION
**Pull request by carlbordum**

### CODE REVIEW

File: oio_rest/oio_rest/utils/test_support.py [hunk 1]
```diff
@@ -63,16 +160,7 @@ def _initdb():
...
-        [os.path.join(BASE_DIR, '..', 'db', 'mkdb.sh')],
-        env=env,
-        stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-        stdin=subprocess.DEVNULL,
-    ) as proc:
-        sql, errortext = proc.communicate()
-
-    assert proc.returncode == 0, 'mkdb failed:\n\n' + errortext.decode()
-
+    sql = _get_db_setup_sql(settings.DATABASE, settings.DB_USER)
```
Comment (danchr) [inline oio_rest/oio_rest/utils/test_support.py]: 🎉 

This is precisely what we discussed at the meeting — that should make @jgj-magenta-aps's life easier!

File: oio_rest/oio_rest/utils/test_support.py [hunk 2]
```diff
@@ -41,6 +41,103 @@ def psql():
     return psql
 
 
+def _get_db_setup_sql(db_name, db_user):
```
Comment (danchr) [inline oio_rest/oio_rest/utils/test_support.py]: This is indeed quite a mess, as you note, and after reading the code, I do understand why. It's better than the _previous_ mess, though!

File: oio_rest/oio_rest/utils/test_support.py [hunk 3]
```diff
@@ -41,6 +41,103 @@ def psql():
...
+    """Return the postgresql + pl/pgsql code necessary for our database
+    to work.
+
+    Brave souls can find db/mkdb.sh in the git history to see what this
```
Comment (danchr) [inline oio_rest/oio_rest/utils/test_support.py]: Hey, I wrote that script — and you're right! 😁

File: oio_rest/oio_rest/utils/test_support.py (hunk 1 above)
Comment (jgj-magenta-aps) [reply oio_rest/oio_rest/utils/test_support.py]: jgj-magenta-aps is pleased with having such friendly peers

**Pull request merged**